sdks/java/io/hadoop/jdk1.8-tests/src/test/resources/kubernetes/cassandra/LargeITCluster/cassandra-service.yaml

```diff
@@ -0,0 +1,28 @@
+#    Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Done- combined Cassandra-service and Cassandra-statefulset and renamed Cassandra-external-service to cassandra-service-for-local-dev